### DISCUSSION TITLE
Are you still using Dexcom G4?

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2797] [General]
If yes, please explain why.

We need to clean up xDrip to make maintenance a reasonable job.

How inconvenient is it for you to upgrade to G5, G6 or G7?

**Comment by digiexchris** [comment]
There are no more supplies of G4 available in Canada, if that helps. There are some remaining supplies of the G5 available, as far as I know from the last time I checked with them 2 years ago.

**Reply by Navid200** [reply]
Supplies (sensors) are identical to G5.

We have already gone ahead and removed G4.
The current stable release does not have it and there have been no complaints, knock on wood!
